src/connectors/configure/__tests__/connectConfigure-test.js

```diff
@@ -0,0 +1,83 @@
+import jsHelper from 'algoliasearch-helper';
+import connectConfigure from '../connectConfigure';
+
+const createState = state =>
+  jsHelper({ addAlgoliaAgent: () => {} }, '', state).state;
```

Comment:
You probably want to use [`jsHelper.make`](https://github.com/algolia/algoliasearch-helper-js/blob/develop/src/SearchParameters/index.js#L559) instead of instantiating the helper and getting the state.

Reply:
So much so that you use literrally everywhere you use create state (it's a static function)